src/elements/index.tsx

```diff
@@ -0,0 +1,7 @@
+export * from "./Box"
+export * from "./Flex"
+export * from "./Join"
+export * from "./Message"
+export * from "./Separator"
+export * from "./Spacer"
+export * from "./Typography"
```

Comment:
@zephraph - I simplified these exports here and below. This will make things easier to manage.

Reply:
I forgot you could do that, heh.